tests/unit/src/test/scala/tests/codeactions/ConvertToNamedArgumentsSuite.scala

```diff
@@ -0,0 +1,55 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.ConvertToNamedArguments
+import scala.meta.internal.metals.codeactions.ExtractValueCodeAction
+
+class ConvertToNamedArgumentsSuite
+    extends BaseCodeActionLspSuite("convertToNamedArguments") {
+
+  check(
+    "convert-to-named-args-basic",
```

Comment:
```suggestion
    "basic",
```

Don't need `convert-to-named-args` prefix, as it's obvious from the test suite name.
(and we can run only this test by `testOnly tests.codeactions.ConvertToNamedArgumentSuite -- *basic*`)